### PR TITLE
Bug 1121997 - [Loop] Wrong texts with mobile in spanish language.

### PR DESCRIPTION
**Pull request by frsela**

### CODE REVIEW

File: app/locales/loop.es.properties [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # {{serviceName}} será sustituido con el nombre de la app (`Firefox Hello`)
 
 # Common
-tagline1 = La manera más fácil de estar en contacto con gente de todo el mundo.
+tagline1 = Firefox Hello es la manera más fácil de conectar de forma gratuita a través de video.
```
Comment (crdlc) [inline app/locales/loop.es.properties]: Firefox Hello is {{serviceName}}

File: app/locales/loop.es.properties [hunk 2]
```diff
@@ -107,41 +107,41 @@ networkDisconnected = Fallo en llamada por desconexión de red
...
 shareTitle = Compartir enlace
-noProblem1 = ¡No hay problema! Comparte el enlace de abajo y te pueden llamar de vuelta desde un navegador.
+noProblem1 = ¡No hay problema! Crea una sala y compártela con él.
 shareVia = Compartir enlace desde:
 message = Mensaje
 email = Email
 other = Otro
 shareMessage = ¡Haz click en el enlace y responde a la llamada!
-shareRoomURLBySMS = Hola! Únete al chat de vídeo en mi sala de {{serviceName}} {{roomName}}. Abre el link: {{roomURL}} Hablamos pronto!
+shareRoomURLBySMS = ¡Hola!Únete al chat de vídeo en mi sala de {{serviceName}} {{roomName}}. Abre el link: {{roomURL}} ¡Hablamos pronto!
```
Comment (crdlc) [inline app/locales/loop.es.properties]: !Únete, is not needed the whitespace?

File: app/js/helpers/contacts_helper.js [hunk 3]
```diff
@@ -219,7 +219,7 @@
      */
     getParticipantName: function(participant) {
       if (!participant.account) {
-        return Promise.resolve(participant.displayName);
+        return Promise.resolve(_('guestTitle'));
```
Comment (jaoo) [inline app/js/helpers/contacts_helper.js]: What about:
```js
return Promise.resolve(participant.displayName || _('guestTitle'));
```
or even:
```js
return Promise.resolve(participant.displayName || _('unknown'));
```

Comment (crdlc) [inline app/js/helpers/contacts_helper.js]: IMHO:

```js
var name = participant.displayName;
return Promise.resolve(!name || name === 'Guest' ? _('guestTitle') : name);
```

**Pull request merged**